src/contracts/test/BalanceOf.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.7.6;
+
+contract BalanceOf {
```

Comment:
Is this used?

Reply:
Yes, for unit tests (instead of deploying a full ERC20 implementation).